src/main/java/com/aetherteam/aether_genesis/data/generators/GenesisLanguageData.java

```diff
@@ -29,23 +30,45 @@ protected void addTranslations() {
         this.addBlock(GenesisBlocks.SKYROOT_WOOD_WALL, "Skyroot Wood Wall");
         this.addBlock(GenesisBlocks.STRIPPED_SKYROOT_WOOD_WALL, "Stripped Skyroot Wood Wall");
 
+
+        this.addBlock(GenesisBlocks.AETHER_CRAFTING_TABLE, "Skyroot Crafting Table");
+        this.addBlock(GenesisBlocks.HOLYSTONE_FURNACE, "Holystone Furnace");
+
         this.addItem(GenesisItems.GOLDEN_SWET_BALL, "Golden Swet Ball");
         this.addItem(GenesisItems.DARK_SWET_BALL, "Dark Swet Ball");
         this.addItem(GenesisItems.DARK_GUMMY_SWET, "Dark Gummy Swet");
         this.addItem(GenesisItems.BLUE_SWET_JELLY, "Blue Swet Jelly");
         this.addItem(GenesisItems.GOLDEN_SWET_JELLY, "Golden Swet Jelly");
         this.addItem(GenesisItems.DARK_SWET_JELLY, "Dark Swet Jelly");
+        this.addItem(GenesisItems.ICESTONE_POPROCKS, "Icestone Poprocks");
+        this.addItem(GenesisItems.COCOATRICE, "Cocoatrice");
+        this.addItem(GenesisItems.WRAPPED_CHOCOLATES, "Wrapped Chocolate");
```

Comment:
Localization needs to be renamed from "Wrapped Chocolate" to "Wrapped Chocolates".